VERSION

```diff
@@ -0,0 +1,5 @@
+# This version information is generated by CMakeLists.txt: Editing is futile
```

Comment:
I'm not sure we should check this in. Either it'll get outdated all the time, or we'll be updating it all the time, making the whole exercise weird?

Reply:
It needs updating as part of the release process (to reflect the tag that's been applied). That's hardly "all the time".